pollux/lib/core/src/test/scala/io/iohk/atala/pollux/core/service/PresentationServiceSpecHelper.scala

```diff
@@ -159,4 +170,43 @@ trait PresentationServiceSpecHelper {
       )
     }
 
+    def createAnoncredRecord(
+        pairwiseVerifierDID: DidId = DidId("did:prism:issuer"),
+        pairwiseProverDID: DidId = DidId("did:prism:prover-pairwise"),
+        thid: DidCommID = DidCommID()
+    ): ZIO[WalletAccessContext, PresentationError, PresentationRecord] = {
+      val anoncredPresentationRequestV1 = AnoncredPresentationRequestV1(
+        requested_attributes = Map(
+          "sex" -> AnoncredRequestedAttributeV1(
```

Comment:
Probably, a good idea for refactoring is to create a package or object `AnoncredsV1` and all corresponding domain classes. `RequestedAttribute` etc.

Reply:
Yep good ide